pkgs/development/python-modules/fonttools/default.nix

```diff
@@ -13,19 +13,21 @@
 , sphinx
 , pytestCheckHook
 , glibcLocales
+, setuptools_scm
 }:
 
 buildPythonPackage rec {
   pname = "fonttools";
-  version = "4.30.0";
+  version = "4.33.3";
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner  = pname;
     repo   = pname;
     rev    = version;
-    sha256 = "1y9f071bdl66rsx42j16j5v53h85xra5qlg860rz3m054s2rmin9";
+    sha256 = "jTVGqNgro2QPD5x/RiR6hyYNXnJDqLnFH6RPNv0IjuE=";
+    leaveDotGit=true;
```

Comment:
What for? `leaveDotGit` has reproducibility problems, so I'd prefer to patch out any magic version number resolution instead (we have the information available anyways.